### PR TITLE
Issue#508 firestore rules

### PR DESCRIPTION
**Pull request by ioaNikas**
- [x] New set of firestore rules

### CODE REVIEW

File: firestore.rules [hunk 1]
```diff
@@ -2,41 +2,53 @@ rules_version = '2';
...
   match /databases/{database}/documents {
 
-    // Note: Add a verification of each document ID => {id: ID_SHOULD_BE_EQUAL_TO_PATH}
-
     /// USERS RULES ///
 
     match /users/{userId} {
-    	allow read, update: if isSignedIn(); // TODO: Secure it as soon as we got Inivitations collection
-    	allow create: if userId() == incomingData().uid;
+    	allow read, update: if isSignedIn();
```
Comment (lsenta) [inline firestore.rules]: I'm pretty sure we still have a security issue, (I can update any user if I'm signed in) keep the TODO

File: firestore.rules [hunk 2]
```diff
@@ -2,41 +2,53 @@ rules_version = '2';
...
+    	allow create: if (userId() == incomingData().uid && userId() == userId)
+      	|| isOrgAdmin(userId(), userOrgId());
     }
 
     /// NOTIFICATIONS RULES ///
 
     match /notifications/{notificationId} {
-    	allow read: if currentUser().uid == existingData().userId;
+    	allow read: if (currentUser().uid == existingData().userId)
```
Comment (lsenta) [inline firestore.rules]: `currentUser().uid ` is `userId()`

File: firestore.rules [hunk 3]
```diff
@@ -2,41 +2,53 @@ rules_version = '2';
...
+        || (userOrgId() == existingData().organizationId)
+      	&& (existingData().state != incomingData().state);
     }
 
     /// ORGANIZATION RULES ///
 
     match /orgs/{orgId} {
-    	allow create: if isSignedIn();
-      allow read, update, delete: if isOrgAdmin(userId(), orgId);
+    	allow create, read: if isSignedIn();
```
Comment (lsenta) [inline firestore.rules]: any user can read any org

File: firestore.rules (hunk 1 above)
Comment (ioaNikas) [reply firestore.rules]: Indeed. We'll remove the TODO and secure it when the invitations will trigger writing of `organizationId` into `user` document.

File: firestore.rules (hunk 3 above)
Comment (ioaNikas) [reply firestore.rules]: Shouldn't we let user read organizations if they want to find / join one ?

**Comment by ioaNikas** [comment]
Duplicate with #1342

**Pull request closed without merging**